.ci/docker/common/install_conda.sh

```diff
@@ -58,7 +58,10 @@ fi
   # as_jenkins conda update -y -n base conda
 
   # Install correct Python version
-  as_jenkins conda create -n py_$ANACONDA_PYTHON_VERSION -y python="$ANACONDA_PYTHON_VERSION"
+  # Also ensure sysroot is using a modern GLIBC and GLIBCXX to match system compilers
```

Comment:
This just updates GLIBC. Nothing to do with GLIBCXX
```suggestion
  # Also ensure sysroot is using a modern GLIBC to match system compilers
```